packages/javascript-sdk/src/oauth2-client/index.ts

```diff
@@ -170,10 +170,16 @@ abstract class OAuth2Client {
       throw new Error('Access token not found in response');
     }
 
+    if (!responseObject.expires_in) {
+      throw new Error('Expires in not found in response');
```

Comment:
I guess it may be over the top, since having no expires_in just means that we don't know when they expire, so we can't pro-actively refresh them (as before this change). I'll look into alternatives 👍